entries/normalizer.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0"?>
+<entry name="normalizer" type="method" return="String">
+	<title>normalizer</title>
+	<desc>Change/transform the value of the element before validating.</desc>
+	<longdesc>
+		The user can change the value of an element before validating the element in question. The new value will be then used by the associated methods instead of the real one.
+		<p>This method should return a String value, otherwise it will throw a TypeError exception.</p>
+		<p>Note that this method desn't change the element's value, it's only change the value passed to the plugin.</p>
```

Comment:
Desnt -> doesnt
Element's -> elements'
It's -> it
Change -> changes
Passed to the plugin -> used for validation